src/Main.elm

```diff
@@ -484,17 +484,27 @@ viewDigits n =
                 Nothing ->
                     0
 
+        addPositiveOrNegativeSign =
+            if n < 0 then
+                Bitmap.negativeSign
+            else                
+                Bitmap.positiveSign
+
         children =
-            String.split "" str
-                |> List.map (toInt >> Bitmap.forInt >> digit >> (\c -> c [] []))
+            (String.split "" str
+                |> List.map (toInt >> Bitmap.forInt >> digit >> (\c -> c [] []))) 
+
+        addNumericalSign =
+            digit addPositiveOrNegativeSign |> (\c -> c [] [])
```

Comment:
just wanted to explain something here. 

you copied `(\c -> c [] [])` from above. in the above case, it's a list of things, so we're `map`ping them into this function. but in this case, you just have a single item.

```diff
  addNumericalSign =
-     digit addPositiveOrNegativeSign |> (\c -> c [] [])
+     (digit addPositiveOrNegativeSign) [] []